resources/views/back/managers/_partials/context-menu.blade.php

```diff
@@ -9,8 +9,29 @@
             <a href="{{ $manager->route('edit') }}" class="block p-3 --link-with-bg">Aanpassen</a>
         @endif
 
-        @include('chief::back.managers._partials.publish-option')
-        @include('chief::back.managers._partials.archive-delete-option')
+        @if($manager->can('update'))
+            @if($manager->model() instanceof \Thinktomorrow\Chief\States\State\StatefulContract)
+                @foreach(\Thinktomorrow\Chief\States\PageStatePresenter::fromModel($manager->model())->transitions() as $transition)
+                    @include('chief::back.managers._transitions.'.$transition)
+                @endforeach
+            @else
+                @if($manager->isAssistedBy('archive'))
+                    @if($manager->assistant('archive')->isArchived())
+                        @include('chief::back.managers._transitions.unarchive')
+                    @else
+                        @include('chief::back.managers._transitions.archive')
+                    @endif
+                @endif
+
+                @if($manager->isAssistedBy('publish'))
+                    @if($manager->assistant('publish')->isPublished())
+                        @include('chief::back.managers._transitions.unpublish')
+                    @else
+                        @include('chief::back.managers._transitions.publish')
+                    @endif
+                @endif
```

Comment:
Het zou cool zijn als we hier enkel moeten loopen over alle assistants en dat de assistants zelf verantwoordelijk zijn voor het renderen van deze views moesten deze van toepassing zijn?
Dat zou ook beperken waar we aanpassingen moeten doen moest er een assistant bijkomen hier.

Reply:
ja moeten we doen!

kunnen we als apart puntje aanpakken na deze PR?